ntbs-ui-tests/Helpers/ConfigHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.IO;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+
+namespace ntbs_ui_tests.Helpers
+{
+    public class ConfigHelper
+    {
+        public static IConfigurationRoot GetConfigurationRoot()
+        {
+            var outputPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            return new ConfigurationBuilder()
+                .SetBasePath(outputPath)
+                .AddJsonFile("testSettings.json", optional: true)
+                .AddUserSecrets("62C1600C-4252-429E-980B-0153542954A7")
+                .AddEnvironmentVariables()
+                .Build();
```

Comment:
What is going on here? In particular, what is the User secret and should this be checked in?

Reply:
So essentially what we're doing here is building our config the same way that the application does - first looking in the settings file, then in the user secrets and then in the environment variables. The application does this out of the box, but we have to bake it ourselves in a test project.
The breakdown here is:
* anything that we can check into the codebase goes in `testSettings.json`
* when running locally we use user secrets (see https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-5.0&tabs=windows)
* when running via github actions we use environment variables for the secrets